napari/layers/utils/color_manager.py

```diff
@@ -0,0 +1,222 @@
+from dataclasses import InitVar
+from typing import Optional, Tuple, Union
+
+import numpy as np
+
+from ...utils.colormaps import CategoricalColormap, Colormap, ensure_colormap
+from ...utils.colormaps.color_transformations import (
+    ColorType,
+    normalize_and_broadcast_colors,
+    transform_color_with_defaults,
+)
+from ...utils.colormaps.standardize_color import transform_single_color
+from ...utils.events.dataclass import Property, evented_dataclass
+from ._color_manager_constants import ColorMode
+from ._color_manager_utils import is_color_mapped
+from .layer_utils import guess_continuous, map_property
+
+
+@evented_dataclass(events=True, properties=True)
```

Comment:
```suggestion
@evented_dataclass
```

(those are default now)

Reply:
Thanks! I updated it.